project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.5.5
\ No newline at end of file
+sbt.version=1.7.1
```

Comment:
That's a bit risky, is it possible to do it as a separate change (easier revert if necessary)?

Reply:
Why it's risky? 2.13.8 is not compatible with the sbt version we were using previously. This two are the only changes in the PR, so the revert should be easy

Reply:
Risky because we're doing to significant version changes in one PR. But I didn't know about the Scala compatibility issue, so let's keep it.